app/models/manageiq/providers/embedded_ansible/automation_manager/inventory.rb

```diff
@@ -1,2 +1,12 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Inventory < ManageIQ::Providers::EmbeddedAutomationManager::InventoryRootGroup
+  def self.create_inventory_raw(tower, inventory_name, hosts)
+    miq_org = tower.provider.default_organization
+    tower.with_provider_connection do |connection|
+      connection.api.inventories.create!(:name => inventory_name, :organization => miq_org).tap do |inventory|
```

Comment:
@bzwei Is this a quick process?
What happens if the name already exists, would we have to report that?